CONTRIBUTING.md

```diff
@@ -9,9 +9,9 @@ Please, follow the next process when you work on your subtask:
 1. **Choosing a Task:**
 
    - Choose a task to work on which:
-     - is not marked as completed with a tick.
-     - does not have an issue created.
-     - is not mentioned in the comments.
+     - Is not marked as completed with a tick.
+     - Does not have an issue created.
+     - Is not mentioned in the comments.
```

Comment:
I'm not an English expert, but I don't think capitalizing the words is right

Reply:
@KareemMAX , Its a basic rule, First letter of any sentence is always capital